### ISSUE TITLE
How to adjust overall loss?

### ISSUE THREAD

**Issue by Shakesbeer333** [matterport/Mask_RCNN#2422]
In #1112 it is mentioned that the overall loss is simply the sum of all minor losses. How can I change this to something like loss = alpha * loss1 + beta * loss2 + ...?

**Comment by tortoiseme** [comment]
In the config file you should find LOSS_WEIGHTS parameter, where you could change the weight of each loss term.
